R/mobr.R

```diff
@@ -352,7 +352,7 @@ permute_comm = function(comm, swap, groups=NULL) {
         else if (swap == 'swapN') {
             sp_draws = sapply(plot_ids, function(x)
                               sample(rep(1:S, sp_abu),
-                                     size=Nperm[x],
+                                     size=Ngroup[x],
                                      replace = T))
```

Comment:
Thanks for catching this. I'm kinda of shocked I was able to get sensical results with the bug in the code that I had.